modules/core/src/main/java/org/locationtech/jts/geom/Polygon.java

```diff
@@ -417,10 +417,17 @@ protected int getSortIndex() {
     return Geometry.SORTINDEX_POLYGON;
   }
 
+  private LinearRing normalized(LinearRing ring, boolean clockwise) {
```

Comment:
Good idea for a better method name.  Might be nice to push this up into Geometry

Reply:
This would be another PR, wouldn't it?

Reply:
Yes, so not needed for this PR